source/deployment/deployment.md

```diff
@@ -48,14 +47,37 @@ Typically a "Mattermost unreachable" error message will be displayed warning use
+- **4. Private Network or VPN with MFA plus private Mobile Apps with EMM (TE & EE)** - Deployment in 3) with the addition of an enterprise mobility management solution such as Mobile Iron, Blackberry Dynamics or AirWatch for access to mobile apps in place of VPN clients. 
+- **5. Private Network or VPN with MFA, plus public Mobile Apps with 2FA (EE-only)** - Deployment in 2) plus use of publicly available mobile apps in iTunes or Google Play with the additional of MFA available in Enterprise Edition via single-sign-on providers like Okta, OneLogin, and SAML-based options, or user credentials with the addition of Google Authenticator. Recommended only for enterprises resourced to provide appropriate network security and monitoring for systems exposed to open internet. For non-enterprise deployments, VPN with MFA is recommended (see above).
+- **6. No VPN with MFA (EE-only)** - Similar to 5) with web, desktop and mobile experiences available outside private cloud with MFA enabled via Mattermost Enterprise Edition. Recommended only for enterprises resourced to provide appropriate network security and monitoring for systems exposed to open internet. For non-enterprise deployments, VPN with MFA is recommended (see above).
+- **7. No VPN, no 2FA (TE & EE)** - Recommended only for non-confidential, unimportant conversations where impact of a compromised system is not material. 
+
+### VPN Setup
+
+Mattermost is designed to offer "modern communication behind your firewall". The standard setup intended to be installed within a private network which can offer multiple factors of authentication, including the securing of access to computing devices as well as the ability to secure physical locations. 
+
+If outside access is required, a virtual private network client (VPN), such as [OpenVPN](https://openvpn.net/), with additional authentication used to connect to Mattermost for web, desktop and mobile experiences. 
+
+### Non-VPN Setup 
+
+If a business owner decides Mattermost should be accessible from the open internet, by allowing unauthenticated access to ports such 80 and 443, the following is recommended: 
+
+1. An IT admin should be assigned to set up appropriate network security, subscribe to [the Mattermost security bulletin](https://about.mattermost.com/security-bulletin/) and [apply new security updates](https://docs.mattermost.com/administration/upgrade.html). 
+2. The organization upgrades to [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/) to enable Google Authenticator, OneLogin, Okta, SAML SSO or similar for at least a two-factor authentication system when three-factor is not appropriate. 
```

Comment:
Similar to an above comment, this should probably be "to enable SAML single sign-on or enable MFA using Google Authenticator". The comment about two-factor vs three-factor also seems unnecessary